javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -690,7 +687,9 @@ public T get(int index) {
         if ((index < 0) || (index >= length())) {
```

Comment:
minor: in line 669 is an unnecessary newline.
IntelliJ IDEA does not preserve your format of the ternary operator:
```java
         final Vector<T> results = ofAll(iterator().filter(predicate));
         return (results.length() == length()) ? this
                                               : results;
```

After formatting the file with our presets, the lines look like this:

```java
        return (results.length() == length()) ? this
                : results;
```

Please format ternary operators like this:

```java
        return (results.length() == length()) ? this : results;
```